packages/cli/test/index.test.ts

```diff
@@ -22,13 +23,18 @@ async function prepare(caseFolderName: string) {
 
   const actualPath = nodepath.join(rootDir, "actual")
   const expectedPath = nodepath.join(rootDir, "expected")
+  const existingPath = nodepath.join(rootDir, "existing")
 
   await fs.rm(actualPath, {
     recursive: true,
     force: true,
   })
 
-  return { rootDir, actualPath, expectedPath }
+  if (glob.sync(existingPath).length === 1) {
+    await fs.cp(existingPath, actualPath, { recursive: true })
+  }
+
+  return { rootDir, actualPath, existingPath, expectedPath }
```

Comment:
since now its implemented for all test cases, remove

```ts
      await fs.cp(
        nodepath.join(rootDir, "existing"),
        nodepath.join(rootDir, "actual"),
        { recursive: true }
      )
```

from this spec:

```
it("should extract and clean obsolete", 
```